tools/convert_datasets/tao/merge_coco_with_lvis.py

```diff
@@ -0,0 +1,106 @@
+import argparse
+import itertools
+import json
+import logging
+from pathlib import Path
+
+import numpy as np
+import pycocotools.mask as mask_util
+from pycocotools.coco import COCO
+from script_utils.common import common_setup
+from tqdm import tqdm
+
+
+def main():
+    # Use first line of file docstring as description if it exists.
+    parser = argparse.ArgumentParser(
+        description=__doc__.split('\n')[0] if __doc__ else '',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '--lvis', type=Path, requirged=True, help='lvis json path')
```

Comment:
`requirged` typo
you need to pull origin master to use the newest precommit config